webpack/scenes/RedHatRepositories/index.js

```diff
@@ -98,11 +107,13 @@ RedHatRepositoriesPage.propTypes = {
   enabledRepositories: PropTypes.shape({
     loading: PropTypes.bool,
     search: PropTypes.shape({}),
+    missingPermissions: PropTypes.array,
```

Comment:
if `missingPermissions` can be only strings, you can be more explicit 
`PropTypes.arrayOf(PropTypes.string)`